RELEASE.md

```diff
@@ -1,20 +1,20 @@
 # Release
 
-To release new version of Kuma follow the steps:
+To release a new version of Kuma follows the steps:
 
 1. Create a CHANGELOG.md.
-2. Double-check with changelog that all new features are documented on kuma.io website.
+2. Double-check with a changelog that all new features are documented on the kuma.io website.
```

Comment:
I think, `with the changelog` might be the right version.

We could also provide a link to the GitHub repo of kuma.io. E.g.,`documented on the [kuma.io website](https://github.com/Kong/kuma-website)`

Reply:
Corrected